src/activities/Association.js

```diff
@@ -33,24 +33,26 @@ export class AssociationEntity extends Entity {
 	/**
 	 * Deletes the association
 	 */
-	async deleteAssociation() {
-		if (!this._entity || !this._entity.hasActionByName(Actions.associations.deleteAssociation)) {
+	async deleteAssociation({ defer = false }) {

```

Comment:
Are there more options being added to the params? Updating to `deleteAssociation( defer = false)` won't require a  change from the caller.

Reply:
Yes, this is by design.

The usages will just need to set `defer = true`, then call `applyDeferredAssociations` to apply the changes.

Reply:
Ok. Will need to make sure we update `d2l-activities` since this will break those `delete/createAssociation` calls if the `siren-sdk` version gets updated.

Reply:
Why?